video_worker/config.py

```diff
@@ -37,3 +50,36 @@ def run(self):
             except yaml.YAMLError as exc:
                 logger.error('Config YAML read error')
                 return
+
+    def _configure(self):
+        """
+        Prompt user for settings as needed for yaml
+        """
+        with open(self.instance_yaml, 'r') as stream:
+
+            try:
+                config_dict = yaml.load(stream)
+            except yaml.YAMLError as exc:
+                logger.error('default YAML read error')
+                return
+
+        output_dict = {}
+
+        for j, k in config_dict.iteritems():
+            sys.stdout.write('\r')
+            new_value = raw_input('%s :' % (j))
+            # clean tailing slashes
+            if new_value is not None and len(new_value) > 0 and new_value[-1] == '/':
+                output_dict[j] = new_value[:-1]
+            else:
+                output_dict[j] = new_value
+                sys.stdout.flush()
+
+        with open(self.instance_yaml, 'w') as outfile:
+            outfile.write(
+                yaml.dump(
+                    output_dict,
+                    default_flow_style = False
+                )
+            )
+        self.settings_dict = output_dict
```

Comment:
My only concern is this file, i did't get the purpose of this change.

Reply:
@muzaffaryousaf This was replaced from [PR#14](https://github.com/edx/edx-video-worker/pull/14) in an effort to green the test, but now that I look at it, it should probably be deleted with the test. Will update